newIDE/app/src/UI/CustomSvgIcons/SquaredChevronArrowDown.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import SvgIcon from '@material-ui/core/SvgIcon';
+
+export default React.memo(props => (
+  <SvgIcon {...props} width="16" height="16" viewBox="0 0 16 16" fill="none">
+    <rect
+      x="0.5"
+      y="0.5"
+      width="15"
+      height="15"
+      rx="3.5"
+      stroke="currentColor"
+      fill="none"
+    />
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
The `...-rule` attributes must be changed to `...Rule`

Reply:
(This is a requirement for React that mostly uses camelCase HTML attributes where HTML uses kebab-case)

Reply:
You have to check for kebab-case attributes each time you use a SVG